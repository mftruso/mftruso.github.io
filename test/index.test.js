import { describe, test, beforeEach, expect, shallowMount } from 'vitest'
import index from '@/pages/index'

describe('index', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(index)
  })

  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  test('Matches Snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
