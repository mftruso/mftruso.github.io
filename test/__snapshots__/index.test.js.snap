// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index Matches Snapshot 1`] = `
"<v-row-stub tag=\\"div\\" align=\\"center\\" justify=\\"center\\">
  <v-col-stub cols=\\"12\\" tag=\\"div\\">
    <article>
      <section>
        <h2 class=\\"headline\\">
          About
        </h2>
        <p>Howdy! I'm a Software Developer based in St. Paul, MN.</p>
      </section>
      <v-spacer-stub></v-spacer-stub>
      <section>
        <h2 class=\\"headline\\">
          Projects
        </h2>
        <p>Here are a few things I've been working on in my spare time.</p>
        <v-row-stub tag=\\"div\\"></v-row-stub>
      </section>
    </article>
  </v-col-stub>
</v-row-stub>"
`;
